src/emulator/apphosting/index.ts

```diff
@@ -12,24 +12,23 @@ interface AppHostingEmulatorArgs {
  * environment for testing App Hosting features locally.
  */
 export class AppHostingEmulator implements EmulatorInstance {
-  private logger = EmulatorLogger.forEmulator(Emulators.APPHOSTING);
   constructor(private args: AppHostingEmulatorArgs) {}
 
   async start(): Promise<void> {
-    this.logger.logLabeled("INFO", Emulators.APPHOSTING, "starting apphosting emulator");
+    logger.logLabeled("INFO", Emulators.APPHOSTING, "starting apphosting emulator");
```

Comment:
q: why did we decide not to use the emulator logger? I assumed that would be the preferred/consistent way for the emulator.

Reply:
We are using the emulator logger. Just refactored so that we don't have to keep instantiating the emulator logger in every file we want to log: https://github.com/firebase/firebase-tools/pull/7781/files/eddd7aed4d0ac0a20b5bf56373c79b8257fc1187#diff-5c4c65d13162d36f52f131c06e42731e0ee91acf7064fc5d2a9568ccb2f4a799R6